src/ast/CommentDecl.ts

```diff
@@ -11,8 +11,19 @@ export default class CommentDecl extends AstNode {
     comments: string[];
 
     public parse(context: Tokenizer): any {
-        // TODO: implement the rest.
+        // We start by checking "[" here, since when we call this, we would have consumed the "comment" token already
+        context.getAndCheckNext("\\[");
         this.comments = [];
+        while (!context.checkToken("\\]")) {
```

Comment:
Maybe we could also check token isn't NO_MORE_TOKENS here too (if we run into this here, we have a problemo)

Reply:
I think it'll be okay without it here. The next `context.getAndCheckNext("\"");` should catch it